### PR TITLE
Wdb 134/harden stream endpoint

### PR DESCRIPTION
**Pull request by kibartas**
Simple PR, fixes the "stream" endpoint not being authorized. PR implements manual JWT validation.

### CODE REVIEW

**Review by tmaciulis22 — APPROVED** [review]
❤️

**Review by povilassta — APPROVED** [review]
Good stuff

File: backend/RestAPI/Controllers/VideosController.cs [hunk 1]
```diff
@@ -142,16 +146,34 @@ public async Task<IActionResult> GetThumbnail(Guid videoId)
...
 
         [HttpGet, Route("stream"), AllowAnonymous]
-        public async Task<ActionResult> Stream(Guid videoId, Guid userId)
+        public async Task<ActionResult> Stream(Guid videoId, string token)
         {
-            if (videoId == Guid.Empty)
+            if (String.IsNullOrWhiteSpace(token))
             {
-                return BadRequest("Video id is not valid");
+                return Unauthorized("No authorization token supplied");
```
Comment (AArlauskas) [inline backend/RestAPI/Controllers/VideosController.cs]: *provided

**Review by AArlauskas — APPROVED** [review]
One naming suggestion, but overall a very good job!

**Pull request merged**